lib/routers/user.js

```diff
@@ -18,5 +18,6 @@ router.use((req, res, next) => {
 });
 
 router.use(require('./profile'));
+router.use('/tasks', require('./tasks'));
```

Comment:
do we ever want to be able to get _another_ user's tasks? if so we should mount in profile router

Reply:
As I understand it - if a task is relevant to you it should be in your task list - I'm not sure if viewing other peoples task lists is a thing?

Reply:
I am also not sure if it is a thing. I was more thinking of a super user having visibility over tasks assigned to a different user, and using permissions middleware to decide if they are able to request tasks for the given profile

Reply:
ok let's change it later if the requirement to do that comes up.